DoSomethingArchive/legacy-website
# Vagrant solr

#### What's this PR do?
-  Installs a Solr 4.7 multicore setup on dev Vagrant boxes that's identical to the one used in all other environments
- Uses Tomcat 7 instead of Tomcat 6
#### Where should the reviewer start?

The most significant changes are in the salt provisioning.  The old solr config files have been removed and replaced with solr config files in a gist that work for a Drupal Solr 4.7 installation.  Tomcat 7 is also installed instead of Tomcat 6.  The changes to the tomcat.sls salt file will give a good overview of what needs reviewing.
#### How should this be manually tested?
- Pull down the code, vagrant destroy your current environment, vagrant up to make sure all Salt changes have happened.
- After adding `127.0.0.1    solr.dev.dosomething.org` to your `/etc/hosts` file, check that you can visit solr.dev.dosomething.org:8080/solr and that all of the affiliates solr instances are listed in the drop down.
#### Any background context you want to provide?

Dev boxes currently are set up to connect to the QA Solr instance, but in the case that we actually need to debug search it's better to have a solr instance set up locally.

cc: @mshmsh5000 @aaronschachter
